src/test/java/org/frankframework/frankdoc/wrapper/TestUtil.java

```diff
@@ -114,7 +106,8 @@ public static String getTestFile(URL url, String charset) throws IOException {
 		if (url == null) {
 			return null;
 		}
-		return readLines(getCharsetDetectingInputStreamReader(url.openStream(), charset));
+		InputStreamReader isr = new InputStreamReader(new BufferedInputStream(url.openStream()), "UTF-8");
```

Comment:
Ik denk dat het mooier is de `Charset` klasse te gebruiken, deze is onderdeel van de JDK dus er is geen dependency voor nodig.

Reply:
Ik heb de Charset class in gebruik genomen. Is het nu goed?